crates/test_utils/wasm/wasm_workspace/multiple_calls/src/lib.rs

```diff
@@ -37,3 +37,15 @@ fn get_entry_multiple(n: TestInt) -> ExternResult<TestBytes> {
 
     Ok(TestBytes(bytes))
 }
+
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize, SerializedBytes)]
+pub struct TwoInt(pub u32, pub u32);
```

Comment:
This probably isn't being used

```suggestion
```

Reply:
Why? It's used in the function below though